src/CodeGen/ODataT4CodeGenerator.cs

```diff
@@ -713,7 +713,8 @@ public bool ModelHasInheritance
                 Debug.Assert(this.EdmModel != null, "this.EdmModel != null");
                 this.modelHasInheritance = this.EdmModel.SchemaElementsAcrossModels()
                             .OfType<IEdmStructuredType>().Any(t => !t.FullTypeName().StartsWith("Org.OData.Authorization.V1") &&
-                             !t.FullTypeName().StartsWith("Org.OData.Capabilities.V1") && t.BaseType != null);

```

Comment:
@xuzhg we removed the basetype null check?

Reply:
Yes. Skip it in the Code Generator.